dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/PropertyUtils.java

```diff
@@ -48,7 +51,12 @@ private PropertyUtils() {
     public static synchronized void loadPropertyFile(String... propertyFiles) {
         for (String fileName : propertyFiles) {
             try (InputStream fis = PropertyUtils.class.getResourceAsStream(fileName);) {
-                properties.load(fis);
+                Properties subProperties = new Properties();
+                subProperties.load(fis);
+                subProperties.forEach((k, v) -> {
+                    logger.info("Get property {} -> {}",  k, v);
```

Comment:
Note that there might be sensitive configs such as access key / access id in the `common.properties` file

Reply:
Yes, do we have any way to mask this, can we create a mask key list, if the config key in the list, then we didn't print the value?

Reply:
Yes but we have to maintain a list of credential keys. I'd like to know why we want to print these properties?

Reply:
Sometimes, I suspect that we didn't load the correct configuration file, so I want to print the config here :)

Reply:
Thanks for your suggestion, I changed the log level to debug.